### PR TITLE
air: init at 1.15.1

### PR DESCRIPTION
**Pull request by Gonzih**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/masgit push --set-upstream origin go-air-initter/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/air/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  };
+
+  vendorSha256 = "0k28rxnd0vyb6ljbi83bm1gl7j4r660a3ckjxnzc2qzwvfj69g53";
+
+  subPackages = [ "." ];
+
+  meta = with lib; {
+    description = "Live reload for Go apps";
+    homepage = "https://github.com/cosmtrek/air";
+    license = licenses.gpl3;
```
Comment (fabaff) [inline pkgs/development/tools/air/default.nix:21]: Upstream doesn't mention that they support "any later" GPLv3 licenses. 

```suggestion
    license = licenses.gpl3Only;
```

Just to be sure https://github.com/cosmtrek/air/issues/142

**Review by fabaff — COMMENTED** [review]
Nitpicking: Could you please update the commit messages to `air: init at 1.15.1`? See [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md)

File: pkgs/development/tools/air/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "air";
+  version = "1.15.1";
+
+  src = pkgs.fetchFromGitHub {
```
Comment (fabaff) [inline pkgs/development/tools/air/default.nix:7]:
```suggestion
  src = fetchFromGitHub {
```

**Comment by Gonzih** [comment]
@fabaff applied changes suggested, thanks for the feedback!

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 114201` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>air</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] package built (see https://github.com/NixOS/nixpkgs/pull/114201#issuecomment-785858000)
- [ ] all depending packages build
- [x] tested on x86_64

```bash
...ew/pr-114201-1]$ ./results/air/bin/air -c test

  __    _   ___  
 / /\  | | | |_) 
/_/--\ |_| |_| \_  // live reload for Go apps, with Go 

2021/02/25 13:27:08 open test: no such file or directory
[...]
```

Looks good to me.

**Pull request merged**